marklogic-data-hub/src/main/java/com/marklogic/hub/step/impl/MappingStepDefinitionImpl.java

```diff
@@ -42,14 +40,7 @@ public MappingStepDefinitionImpl(String name) {
 
         options.put("sourceDatabase", HubConfig.DEFAULT_STAGING_NAME);
         options.put("targetDatabase", HubConfig.DEFAULT_FINAL_NAME);
-        Versions versions = ApplicationContextReference.getBean(Versions.class);
-        if (versions == null || versions.isVersionCompatibleWithES()){
-            setModulePath("/data-hub/5/builtins/steps/mapping/entity-services/main.sjs");
-        }
-        else {
-            setModulePath("/data-hub/5/builtins/steps/mapping/default/main.sjs");
-        }
-
```

Comment:
Spoke with @sbayatpur , she said DHF 5.3.0 will be supported on ML servers  >=9.0-11 for 9.x and >=10.0-2.1 for 10.x. Hence removed the check.